### ISSUE TITLE
Corrupt quality values

### ISSUE THREAD

**Issue by mbassalbioinformatics** [lmdu/pyfastx#44]
Hi

Ive got an unaligned BAM file from a pac bio long-read run which ive converted to a fq using bedtools. I am then trying to use pyfastx to play around with the file and ive noticed something (that could be my own fault).

When i load the fq.gz file from within python, pufastx will open the file, index it and then i can start calling the stats on it.
If however i use the commandline argument then it will index fine, but when i try to run the stats will give the following error

```
Traceback (most recent call last):
  File ".../bin/pyfastx", line 8, in <module>
    sys.exit(main())
  File ".../lib/python3.9/site-packages/pyfastxcli.py", line 578, in main
    args.func(args)
  File ".../lib/python3.9/site-packages/pyfastxcli.py", line 99, in fastx_info
    fq.maxlen, fq.minlen, fq.maxqual, fq.minqual, ",".join(fq.encoding_type)]
TypeError: Quality values corrupt. found [34, 126] where [33, 104] was expected
```

Any ideas what might be going on?

**Comment by lmdu** [comment]
I am sorry, the pyfastx could not support fastq file with qulity score between 34 and 126. I will extend it in new version.

**Comment by lmdu** [comment]
We have fixed this issue in new versions >= 0.9.0
